var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -143,6 +143,8 @@ class Neovim(CMakePackage):
         depends_on("cmake@3.13:", type="build")
         depends_on("libvterm@0.3.3:")
         depends_on("tree-sitter@0.20.9:")
+    with when("@master"):
```

Comment:
```suggestion
    with when("@master:"):
```

Reply:
You’re actually right, it’s a nonsense practice I was used to, because it would be an easy replacement with the next release.